packages/esm-patient-list-app/src/patient-list-list/patient-list-table.component.tsx

```diff
@@ -44,8 +44,8 @@ interface PatientListTableContainerProps {
   handleCreate?: () => void;
   error: Error;
   isValidating: boolean;
-  searchString: string;
-  setSearchString: (searchString) => void;
+  searchTerm: string;
+  handleSearch?: (searchString: string) => void;
```

Comment:
`handleSearch` should be a required prop, since it's critical for handling search